lab-allie/app/view/landing/landing-controller.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// require('./_landing.scss');
+
+module.exports = ['$log',
+  '$location',
+  // '$rootScope',
+  // 'authService',
+  function($log, $location) {
+  // this.$onInit = () => {
```

Comment:
Definitely want that `$onInit()` active for Angular to initialize your controller logic.